packages/rfw/coverage/lcov.info

```diff
@@ -0,0 +1,3519 @@
+SF:lib/src/dart/binary.dart
```

Comment:
this file should not be checked in. you can add it to the .gitignore if that helps.

Reply:
Removed.